code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -722,11 +725,13 @@
 	restore_eyesight(prev_affected_mob, eyes)
 
 /datum/reagent/medicine/oculine/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = FALSE
 	var/normalized_purity = normalise_creation_purity()
 	affected_mob.adjust_temp_blindness(-4 SECONDS * REM * seconds_per_tick * normalized_purity)
 	affected_mob.adjust_eye_blur(-4 SECONDS * REM * seconds_per_tick * normalized_purity)
 	var/obj/item/organ/internal/eyes/eyes = affected_mob.get_organ_slot(ORGAN_SLOT_EYES)
 	if(eyes)
+		. = TRUE
```

Comment:
You `return ..()` further below so I think this one actually didn't do anything.
However can I suggest:
```
var/obj/item/organ/internal/eyes/eyes = affected_mob.get_organ_slot(ORGAN_SLOT_EYES)
if (isnull(eyes)
	return ..()
<rest of proc>

return ..() || TRUE
```